src/execution/sql/functions/string_functions.cpp

```diff
@@ -447,4 +454,55 @@ void StringFunctions::Chr(StringVal *result, exec::ExecutionContext *ctx, const
     }
   }
 }
+
+void StringFunctions::Md5(StringVal *result, exec::ExecutionContext *ctx, const StringVal &str) {
+  if (str.is_null_) {
+    *result = StringVal::Null();
+    return;
+  }
+  char new_str[str.GetLength()];
+  memcpy(new_str, str.GetContent(), str.GetLength());
+  MD5_CTX md5ctx;
+  MD5Init(&md5ctx);
+  MD5Update(&md5ctx, reinterpret_cast<unsigned char *>(new_str), str.GetLength());
+  unsigned char md5sum[16];
```

Comment:
For the string functions I kept the code from the original PRs because I didn't really want to clobber other peoples code. I think the idea here is that each character in this array gets converted into a string representation of it's hex, which would be two characters. So the final result will be 32 characters.

I think we should probably just remove MD5 from this PR and revisit it individually.